TurkishFilter/sections/specific.txt

```diff
@@ -504,9 +498,10 @@ myburc.com##div[style="height:280px;"]
 canlitv.center###sidebar_reklam
 ||erotik123.com/top.js
 filmifullizle.club##a[href="/filmifullizle.php"]
-/main_tabs*.js$domain=720pizlemek1.com
-/video_ob*.js$domain=720pizlemek1.com
-/ft*.js$domain=720pizlemek1.com
+/s-mi*.js$domain=fullhdfilmcibaba1.com|720pizlemek1.com|fullhdfilmizlett.com
+/main_tabs*.js$domain=fullfilmcidayi1.com|720pizlemek1.com
+/video_ob*.js$domain=fullfilmcidayi1.com|fullhdfilmcibaba1.com|fullhdfilmizlett.com|realfilmizle3.com|720pizlemek1.com
+/ft*.js$domain=|fullfilmcidayi1.com|fullhdfilmizlett.com|realfilmizle3.com|fullhdfilmcibaba1.com|720pizlemek1.com
```

Comment:
`/ft*.js$domain=|fullfilmcidayi1.com|fullhdfilmizlett.com|realfilmizle3.com|fullhdfilmcibaba1.com|720pizlemek1.com` this `|` in `domain=|` is redundant, remove it.